hack/test-port-forwarding.pl

```diff
@@ -20,8 +20,24 @@
 use Socket qw(inet_ntoa);
 use Sys::Hostname qw(hostname);
 
+# check for nc
+sub check_netcat_installed {
+    my $nc_path = `which nc 2>/dev/null`;
```

Comment:
`which` is often missing. `command -v` should work.

Reply:
Done 😄